openmsupply/application-manager-web-app
# #441 part 1 application page LOQ

branched from: #314-duplicate-application-responses
requires back end: #314fe-requires-restart-status

the purpose of this branch was to give an introduction (via the Testing branch) to how work up to this branch can be used to full-fill #441 (application page and application summary for LOQ) and #437 (application home for change requested)

### Changes

* slight change to progress bar, something I've noticed when testing, when anything on a page is not valid we should show it as such (rather then blue circle)

### Testing

use branch #441-TESTING-part-1-application-page-LOQ
and back end branch above

create a review for "ABC123" with LOQ

the go to application home 

* notice that header is showing that 'there are issues'
* notice button you can click to start change requested (this should only be clicked once, after that use resume button)
* press CheckChangesRequested
* Notice progress bar statuses and json under the elements
* play around with changing things and trying to submit (logic should work just like it did previously in non linear, will consider change request that are not done as invalid and incomplete)


Here is the diff for reference: `https://github.com/openmsupply/application-manager-web-app/compare/%23441-part-1-application-page-LOQ...%23441-TESTING-part-1-application-page-LOQ`
